### ISSUE TITLE
Tweet Links and Media Files

### ISSUE THREAD

**Issue by BrandonKMLee** [bisguzar/twitter-scraper#188]
As observed in https://github.com/taspinar/twitterscraper#1-motivation it seemed that Tweet Links and Tweet Multimedia files can be acquired. Is there a way of getting them without using taspinar's repo?

**Comment by Prit2341** [comment]
Yes, there are other ways to acquire tweet links and multimedia files from Twitter without using the taspinar/twitterscraper One way is to use the Twitter API to access tweets and retrieve the information you need. The API allows you to search for tweets, retrieve user timelines, and access various other data. Additionally, you can use Python libraries such as Tweepy or python-twitter to interact with the API and access tweets. You could also scrape tweets using a web scraping library such as Beautiful Soup or Scrapy.

**Comment by Prit2341** [comment]
If you Like I want to work on this

**Comment by BradKML** [comment]
Might find a way to see if BeautifulSoup download combo can be made. There should be other solutions that can be chained up.
https://github.com/JustAnotherArchivist/snscrape/issues/143 (slightly convoluted)
https://github.com/istudyatuni/artworks-downloader/blob/master/art_dl/sites/twitter.py (relies on nitter gateway, might not get everything)
